### PR TITLE
Modeling rules for Taniumm Threat Response.

### PR DESCRIPTION
**Pull request by yasta5**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-3127

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

### CODE REVIEW

File: Packs/TaniumThreatResponse/README.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Tanium Threat Response
+This pack includes Cortex XSIAM content.
+
+
+## Configuration on Server Side
+You need to configure a Socket Receiver on Tanium side.
```
Comment (ShirleyDenkberg) [inline Packs/TaniumThreatResponse/README.md:6]:
```suggestion
You need to configure a Socket Receiver on the Tanium side. 
```

File: Packs/TaniumThreatResponse/README.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
+# Tanium Threat Response
+This pack includes Cortex XSIAM content.
+
+
+## Configuration on Server Side
+You need to configure a Socket Receiver on Tanium side. 
+
+Follow the below steps to configure the socket receiver:
```
Comment (ShirleyDenkberg) [inline Packs/TaniumThreatResponse/README.md:8]:
```suggestion
Perform the following steps to configure the Socket Receiver:
```

File: Packs/TaniumThreatResponse/README.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+1. Go to **Modules** > **Connect**.
+2. Enter a name and description for the connection.
+3. Select **Event** from the dropdown under **Source**.
+4. Select **Tanium Threat Response** from the dropdown under **Event Group**.
+5. Select **Match Alerts Raw**.
+6. Select **Socket Receiver** from the dropdown under **Destination**.
+7. Specify a unique name for the **Destination Name**.
+8. Under **Host** fill in the name or the ip address of the siem.
```
Comment (ShirleyDenkberg) [inline Packs/TaniumThreatResponse/README.md:16]:
```suggestion
8. Under **Host**, fill in the name or the IP address of the SIEM.
```

File: Packs/TaniumThreatResponse/README.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+This pack includes Cortex XSIAM content.
+
+
+## Configuration on Server Side
+You need to configure a Socket Receiver on Tanium side. 
+
+Follow the below steps to configure the socket receiver:
+1. Go to **Modules** > **Connect**.
+2. Enter a name and description for the connection.
+3. Select **Event** from the dropdown under **Source**.
```
Comment (ShirleyDenkberg) [inline Packs/TaniumThreatResponse/README.md:11]:
```suggestion
3. From the **Source** dropdown, select **Event**.
```

Comment (ShirleyDenkberg) [inline Packs/TaniumThreatResponse/README.md:11]:
```suggestion
3. From the **Source** dropdown, select **Event**.
```

File: Packs/TaniumThreatResponse/README.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+4. Select **Tanium Threat Response** from the dropdown under **Event Group**.
```
Comment (ShirleyDenkberg) [inline Packs/TaniumThreatResponse/README.md:12]:
```suggestion
4. From the **Event Group** dropdown, select **Tanium Threat Response**.
```

File: Packs/TaniumThreatResponse/README.md [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+5. Select **Match Alerts Raw**.
+6. Select **Socket Receiver** from the dropdown under **Destination**.
```
Comment (ShirleyDenkberg) [inline Packs/TaniumThreatResponse/README.md:14]:
```suggestion
6. From the **Destination** dropdown, select **Socket Receiver**.
```

**Comment by ShirleyDenkberg** [comment]
@evisochek Doc review completed.

**Review by evisochek — APPROVED** [review]

**Pull request merged**